### DISCUSSION TITLE
Is there a way to get the CRS data from the shz/shapefile being read with the spatial extension (as some kind of table metadata, etc).

### DISCUSSION THREAD

**Discussion by jdoig** [duckdb/duckdb-spatial#155] [Q&A]
Please excuse me if I'm being stupid (or just haven't read the manual thoroughly enough)... But is there anyway from within duckdb to get the CRS data from the `.prj` file in the shx I'm parsing with ST_READ?

I see that the extension integrates with PROJ which makes me suspect what I'm asking is possible (though I do see from the documentation that the geometry itself can't contain SRID information yet)

**Comment by Maxxen** [comment]
Not yet, I've been meaning to add something like a `st_describe` function to output metadata, or output the CRS info from the dataset as an additional column. Im converting this to an issue so that we can track development of this feature better, might be able to squeeze it in soon.

**Reply by Maxxen** [reply]
Issue #156
